### PR TITLE
Docs: recommend to install from packages

### PR DESCRIPTION
**Pull request by Al2Klimov**
## Blocked by

* #287

### CODE REVIEW

**Review by bobapple — APPROVED** [review]
Looks good to me 👍 

If you want to see it live in action, here's the rendered version: https://icinga.com/docs/icinga-web-graphite-integration/docs/

File: doc/06-Demonstration.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
 # <a id="Demonstration"></a>Demonstration
```
Comment (lippserd) [inline doc/06-Demonstration.md:1]: #287 will remove the Dockerfile. Please revert the changes made here as this file is also subject to removal.

**Review by lippserd — CHANGES_REQUESTED** [review]
In addition to the requested changes please omit the `the` in headings, e.g. `Installing the ...`, `Configuring the ...`. This is also the case in our other docs.

File: doc/06-Demonstration.md (hunk 1 above)
Comment (Al2Klimov) [reply doc/06-Demonstration.md:1]: That issue shouldn’t take long to resolve. I'll rebase this one on top if necessary.

File: doc/02-Installation.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+<!-- {% if index %} -->
+# Installing the Icinga Web Graphite Integration
 
+The recommended way is to use prebuilt packages for
+all supported platforms from our official release repository.
+Please follow the steps listed for your target operating system,
+which guide you through setting up the repository and installation.
+
+<!-- {% else %} -->
 ## Requirements
```
[the remark this answers is not in the document]
Comment (lippserd) [reply doc/02-Installation.md:10]: > [...] oh, cool, I've installed everything and then in our config doc you'll be surprised with a _Graphite Web_ URL being needed.

You are absolutely right.

File: doc/06-Demonstration.md (hunk 1 above)
Comment (lippserd) [reply doc/06-Demonstration.md:1]: Ready to rebase.

File: README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ the monitoring module.
 
 ## Documentation
 
-* [Installation](doc/02-Installation.md)
+* [Installation](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/02-Installation/)
 * [Configuration](doc/03-Configuration.md)
 * [Templates](doc/04-Templates.md)
 * [Development](doc/06-Development.md)
```
Comment (nilmerg) [inline README.md:25]: Please let all documentation links in this file point to the website.

File: module.info [hunk 4]
```diff
@@ -4,5 +4,5 @@ Requires:
   Libraries: icinga-php-library (>=0.9.0), icinga-php-thirdparty (>=0.10.0)
   Modules: monitoring (>=2.9.0), icingadb (>=1.0.0)
 Description: Icinga Graphite module
```
Comment (nilmerg) [inline module.info:6]: Use the new product name here.

**Review by nilmerg — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ the monitoring module.
...
 ## Documentation
 
-* [Installation](doc/02-Installation.md)
-* [Configuration](doc/03-Configuration.md)
-* [Templates](doc/04-Templates.md)
-* [Development](doc/06-Development.md)
+* [Installation](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/02-Installation/)
+* [Configuration](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/03-Configuration.md)
+* [Templates](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/04-Templates.md)
+* [Development](https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/06-Development.md)
```
Comment (nilmerg) [inline README.md:25]: Have you tried these links by chance? I guess not.

**Review by nilmerg — CHANGES_REQUESTED** [review]
I don't quite like that the from source section is now buried in the sub directory. The ones reading this on Github are more likely to need that section in the main installation section.

**Comment by lippserd** [comment]
> I don't quite like that the from source section is now buried in the sub directory. The ones reading this on Github are more likely to need that section in the main installation section.

There is a link pointing right to it in installation.md.

**Comment by Al2Klimov** [comment]
I'll first have a look if you can convince Eric to do the same for Cube.

File: README.md (hunk 5 above)
Comment (Al2Klimov) [reply README.md:25]: Isn’t it enough for them to work after release?

Comment (lippserd) [reply README.md:25]: Come on, they are simply wrong:

https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/04-Templates/

Comment (nilmerg) [reply README.md:25]: The work fine already: https://icinga.com/docs/icinga-web-graphite-integration/latest/doc/03-Configuration/

**Comment by nilmerg** [comment]
> > I don't quite like that the from source section is now buried in the sub directory. The ones reading this on Github are more likely to need that section in the main installation section.
> 
> There is a link pointing right to it in installation.md.

Ah, didn't notice that!

File: doc/02-Installation.md [hunk 6]
```diff
@@ -1,15 +1,24 @@
-# Installation
+<!-- {% if index %} -->
+# Installing Icinga Web Graphite Integration
 
-## Requirements
+The recommended way to install this Integration is to use prebuilt
```
Comment (nilmerg) [inline doc/02-Installation.md:4]: `integration` is not a name so write it lowercase please. Although, I would avoid this altogether and write just `It is recommended to use prebuilt...`

File: doc/02-Installation.md [hunk 7]
```diff
@@ -1,15 +1,24 @@
...
+packages for all supported platforms from our official release repository.
+Obviously [Icinga Web](https://icinga.com/docs/icinga-web) itself
```
Comment (nilmerg) [inline doc/02-Installation.md:6]: Starting the sentence with `Obviously` sounds rather harsh to me. Please rephrase that.

File: doc/02-Installation.md (hunk 6 above)
Comment (nilmerg) [reply doc/02-Installation.md:4]: Of course, in `Icinga Web Graphite Integration` it's part of the name and hence the first letter is uppercase. But used individually it's just wrong to do that.

File: doc/02-Installation.md [hunk 8]
```diff
@@ -1,15 +1,24 @@
...
-  * The `monitoring` or `icingadb` module needs to be configured and enabled.
-* Graphite and Graphite Web
+The following steps will guide you through installing
+and setting up Icinga Web Graphite Integration.
+<!-- {% else %} -->
+<!-- {% if not icingaDocs %} -->
+
+## Installing the Package
+
+If the [repository](https://packaages.icinga.com) is not configured yet, please add it first.
```
Comment (nilmerg) [inline doc/02-Installation.md:18]: Here is also still the typo in the subdomain.

**Review by nilmerg — CHANGES_REQUESTED** [review]
I still find it wrong to read `Configuring Icinga Web Graphite Integration` and not `Configuring the Icinga Web Graphite Integration`. It's just better wording.

**Review by nilmerg — APPROVED** [review]

**Review by lippserd — APPROVED** [review]
